### ISSUE TITLE
As a users I want to have support to PHP 7.4.0 in order to run the last version.

### ISSUE THREAD

**Issue by danielcardoso** [MyIntervals/emogrifier#827]
```
Problem 1
    - Installation request for pelago/emogrifier 3.0.0 -> satisfiable by pelago/emogrifier[v3.0.0].
    - pelago/emogrifier v3.0.0 requires php ^5.6.0 || ~7.0.0 || ~7.1.0 || ~7.2.0 || ~7.3.0 -> your PHP version (7.4.0) does not satisfy that requirement.
```

**Comment by rsanaie** [comment]
Yes please!

**Comment by oliverklee** [comment]
Duplicate of #819/#821.

**Comment by oliverklee** [comment]
I have just released version 3.1.0 (including support for PHP 7.4 and Symfony 5) and posted about it on social media.
